src/index.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <title>Day03</title>
   <base href="/">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
this is one way to import bootstrap. but it is better to use npm package since all dependencies are tracked through package.json in angular